docs/src/main/asciidoc/_includes/devtools/extension-add.adoc

```diff
@@ -1,7 +1,7 @@
 [source,bash,subs=attributes+, role="primary asciidoc-tabs-sync-cli"]
 .CLI
 ----
-quarkus extension add '{add-extension-extensions}'
+quarkus extension add {add-extension-extensions}
```

Comment:
Thanks but let's not rush this in as I think it was a bit hard to find a combination that was more or less working for several shells (and OSes).

Reply:
I managed to test the Quarkus Cli using virtual machines 😅 
The conclusion is that the command to add extensions always works without quotes for both the cases of adding one or more extensions, across the 3 operating systems.

In detail:

| Description | with quotes | without quotes |
| ----------- | ------------- | -------------- |
| MacOs Sonoma:  one extension | ✅  | ✅ |
| MacOs Sonoma:  multiple extensions | ❌ | ✅ |
| Ubuntu 22.04.3:  one extension | ✅  | ✅ |
| Ubuntu 22.04.3:  multiple extensions | ❌ | ✅ |
| Windows command line:  one extension | ❌   | ✅ |
| Windows command line:  multiple extensions | ❌ | ✅ |
| Windows powershell: one extension | ✅  | ✅ |
| Windows powershell: multiple extensions | ❌ | ✅ |


Some screenshots:
[quarkus-cli.zip](https://github.com/quarkusio/quarkus/files/12894225/quarkus-cli.zip)